notebooks/116-sparsity-optimization/requirements.txt

```diff
@@ -0,0 +1 @@
+optimum[openvino, nncf]~=1.5
```

Comment:
does it uses nncf 2.3.0?
if yes, i think it can lead to some troubles (e.g. model onnx saving with torch 1.13 not supported) and break some notebooks whiich requires new api...

Please also consider to install requirements inside notebook

Reply:
I am waiting for the upcoming NNCF release.

Reply:
Moved to notebook.